ocaml/testsuite/tests/pattern-guards/test.ml

```diff
@@ -283,28 +283,89 @@ single_bar_syntax None;;
 - : int = 0
 |}];;
 
```

Comment:
This test file is getting very long. Should we consider splitting it up into multiple files? Using more comments to split it up?

```
(***********
 These tests show XYZ
*)
```

Reply:
True, although we may be better-suited to do this on the parsing branch where we have already improved test readability. We could also try to merge the parsing branch and then modify this PR to merge into ocaml-flambda/flambda-backend/pattern-guards instead.